video/out/opengl/context.c

```diff
@@ -23,13 +23,7 @@
 // 0-terminated list of desktop GL versions a backend should try to
 // initialize. The first entry is the most preferred version.
 const int mpgl_preferred_gl_versions[] = {
```

Comment:
Maybe this comment should be reworded to be clearer while we're at it. 320 isn't really the "preferred" version, it's simply that we want a version that is compatible with 320. Heck it might be worth changing to the name of the constant as well to like `mpgl_compatible_gl_versions[]`.

Reply:
IMHO it seems fine as-is, that's why I've left it. Can you suggest any concrete wording for the comment?

Since we have `core` and `compatibility` GL profiles, let avoid `compatible` - `required` might be a better option. After all, we go through a list of required versions, top one being the most preferred one and error if we fail to create/meet any of the requirements.

Reply:
I like the `required` wording. That makes it much clearer imo. How about we change the comment to simply:
``` 
// 0-terminated list of desktop GL versions a backend should try to 
// initialize. The each entry is the minimum required version. 
```

And then change `mpgl_preferred_gl_versions` to `mpgl_required_gl_versions`.

Reply:
Non-native speaker, but `The each entry is the minimum required version` seems rather broken.
Nevertheless - updated PR uses the suggestion verbatum.

Reply:
Apologies, I typo'd that badly and didn't even notice. I should have said: `Each entry is the minimum required version. `